charts/super-agent-deployment/values.yaml

```diff
@@ -113,12 +113,8 @@ config:
     #    headers:
     #      some-key: some-value
     #  agents:
-    #    infra-test:
-    #      agent_type: namespace/com.newrelic.infrastructure_agent:0.0.2
-    #    on-host-collector:
-    #      agent_type: namespace/io.opentelemetry.collector:0.0.1
     #    kubernetes-collector:
-    #      agent_type: namespace/io.k8s.opentelemetry.collector:0.0.1
+    #      agent_type: newrelic/io.opentelemetry.collector:0.0.1
```

Comment:
If we are modifying this to have a working example, I'd change the `agent-id` to `open-telemetry` for now.